src/server.h

```diff
@@ -141,9 +141,8 @@ struct hdr_histogram;
 #define CONFIG_MIN_RESERVED_FDS 32
 #define CONFIG_DEFAULT_PROC_TITLE_TEMPLATE "{title} {listen-addr} {server-mode}"
 #define DEFAULT_WAIT_BEFORE_RDB_CLIENT_FREE                                                                            \
-    60 /* Grace period in seconds for replica main                                                                     \
-          channel to establish psync. */
-#define INCREMENTAL_REHASHING_THRESHOLD_US 1000
+    60                                              /* Grace period in seconds for replica main                        \
+                                                       channel to establish psync. */
```

Comment:
We have fixed the clang-format rules so it doesn't require this strange style anymore. Please go back to the more normal style, like on the other lines above and below:

```
#define DEFAULT_WAIT_BEFORE_RDB_CLIENT_FREE 60 /* Grace period in seconds for replica main \
                                                * channel to establish psync. */
```